generate.py

```diff
@@ -130,8 +128,9 @@ def main(
             top_k=top_k,
         )
         t = time.perf_counter() - t0
+
         print(tokenizer.decode(y))
-        print(f"Time for inference {i + 1}: {t:.02f} sec total, {max_new_tokens / t:.02f} tokens/sec", file=sys.stderr)
```

Comment:
max_new_tokens wasn't correct here because we might have generated less tokens than that